server/db/models/User.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function (sequelize, DataTypes) {
```

Comment:
No so the Role.hasMany(User) relationship adds a role_id to the User table which will specify admin and staff booleans for each user. When we do contractor uploads or they login through the contractor side of the application, it should automatically be set to admin=false. Admin = true should only be associated with the users creating an organization for the first time or set by the original admin manually.